http-client/src/main/java/io/airlift/http/client/spnego/SpnegoAuthentication.java

```diff
@@ -136,6 +155,9 @@ public void apply(Request request)
                 catch (LoginException e) {
                     throw new RuntimeException(format("Failed to establish LoginContext for request %s", request.getURI()), e);
                 }
+                catch (Exception e) {
```

Comment:
We shouldn't need this, since there aren't any other checked exceptions that can occur here. Maybe make this `RuntimeException` if we think there are unchecked exceptions to handle.